priv/priv_test.go

```diff
@@ -53,11 +53,11 @@ func (ts *PrivTestSuite) TestPrivMutex(c *C) {
 	c.Assert(helpers.FileExists(lockfile), Equals, false)
 
 	privMutex := New(lockfile)
-	c.Assert(privMutex, Not(IsNil))
+	c.Assert(privMutex, NotNil)
 	c.Assert(helpers.FileExists(lockfile), Equals, false)
 
 	err := privMutex.Unlock()
-	c.Assert(err, DeepEquals, ErrNotLocked)
+	c.Assert(err, NotNil)
```

Comment:
What kind of error do we get now? Would it make sense to user "ErrorMatches" in this test instead?